src/Tooltip/tooltip.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from "prop-types";
+import { Tooltip } from '@mui/material';
+
+export function PSATooltip({
+  title, 
+  placement = 'bottom', 
+  arrow = true,
+  enterTouchDelay = 0, 
+  dangerouslySetInnerHTML, 
+  tooltipContent, 
+  ...props
+}) {
+  return (
+    <Tooltip
+      title={dangerouslySetInnerHTML ? undefined : title}
+      placement={placement}
+      arrow={arrow}
+      enterTouchDelay={enterTouchDelay}
+      {...props}
+    >
+      {tooltipContent}
```

Comment:
@YazidZidane My understanding was that instead of passing content like this we would pass it in as a property of tooltip and then tooltip would be self closing.